### DISCUSSION TITLE
Setting up as a newbie

### DISCUSSION THREAD

**Discussion by serkankalay** [CaiJimmy/hugo-theme-stack#883] [Q&A] [answered]
Hello everyone,

I loved this theme and wanted to use it to start a blog. I have followed the quick start template and created a repository named blog [here](https://github.com/serkankalay/blog). I want to host it with GitHub Pages, and therefore updated the `baseurl` in the `config.toml` as `"https://serkankalay.github.io/blog"`. However, all I see at https://serkankalay.github.io/blog is a weirdly formatter version of the site. Maybe this is a silly question but I couldn't find anything similar to this. 

Thanks a lot for any help.

**Comment by CaiJimmy** [comment, marked as answer]
Hello there, 

This is really weird. I've had a look at the `gh-pages` branch of your repository, and the generated HTML has a `blog` prefix for all resources. But https://serkankalay.github.io/blog does not (hence all resources, including CSS, return 404).

I have no idea what this could be. I suspect it might be related to the configuration of your site's GitHub pages. The automatic "Deploy to GitHub Pages" action included in the starter template runs successfully. But [pages-build-deployment](https://github.com/serkankalay/blog/actions/workflows/pages/pages-build-deployment) was last triggered 2 days ago. This must be the cause of the site not updating correctly.

This is the configuration of the starter template repository:

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/5889006/5ee54d73-fe85-450e-b2ad-1bf01d212825)

**Reply by serkankalay** [reply]
Hi Jimmy,

Thanks a lot for your reply. Actually, that allowed me to solve it. Below is the summary of what happened if you're interested. But once again thanks a lot for this wonderful theme & template!

The problem I had first was I had the `baseurl` set to `https://serkankalay.github.io` but the repo name was `blog`. So first in an attempt, I renamed the repo to `serkankalay.github.io` and that's when the `pages-build-deployment` was triggered. However, I didn't notice I still had the `blog` suffix. Then, I reverted the repo name back to `blog` thinking that didn't work out and started playing with the deployment settings. I set it to `GitHub Actions`. But after your post I both reverted it back to `gh-pages` `/(root)` and also the `baseurl` to `serkankalay.github.io/blog`.
